velox/functions/prestosql/aggregates/tests/AverageAggregationTest.cpp

```diff
@@ -341,6 +341,48 @@ TEST_F(AverageAggregationTest, avgDecimal) {
       {"avg(c0)"},
       {},
       {makeRowVector({makeShortDecimalFlatVector({3'000}, DECIMAL(10, 1))})});
+
```

Comment:
It seemed like you did the null checks for decodedPartial constant mapping case. Please can you explain what and why your new tests trigger the code paths that had the null checks.

Reply:
The tests trigger initialisation of > 1 grouping sets and accumulators.